ddpui/tests/api_tests/test_pipeline_api.py

```
@@ -47,7 +47,7 @@
 from ddpui.models.org_user import OrgUser
 from ddpui.models.role_based_access import Role, RolePermission, Permission
 from ddpui.models.tasks import DataflowOrgTask, OrgDataFlowv1, OrgTask, Task, TaskLock
-from ddpui.models.org_user import OrgUser, OrgUserRole
+from ddpui.models.org_user import OrgUser
```

Ruff (0.11.9)

Redefinition of unused `OrgUser` from line 47

Remove definition: `OrgUser`

(F811)